massband/diffusion/kinisi_diffusion.py

```
@@ -1,239 +1,266 @@
+            p_high = 100 * st.norm.cdf(s)
+            credible_intervals.append([p_low, p_high])
+
+        # --- MSD plot ---
+        fig, ax = plt.subplots()
+        ax.plot(
+            dt.values,
+            diff.msd.values,
+            "k-",
+            label=f"D = {d_mean:.2e} ± {d_std:.2e} {diff.D.unit}",
+        )
+        for i, (ci, s) in enumerate(zip(credible_intervals, sigmas)):
```

Ruff (0.12.2)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)